core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitConfig.java

```diff
@@ -17,6 +17,13 @@
 
   private SerializableStrategy strategy;
 
+  // for two-phase consensus commit
+  public static final String TWO_PHASE_CONSENSUS_COMMIT_PREFIX = PREFIX + "2pcc.";
+  public static final String MANAGE_ACTIVE_TRANSACTIONS =
+      TWO_PHASE_CONSENSUS_COMMIT_PREFIX + "manage_active_transactions";
```

Comment:
```suggestion
      TWO_PHASE_CONSENSUS_COMMIT_PREFIX + "active_transactions_management.enabled";
```
`.enabled` is common for boolean? (not fully sure but I used it for Scalar DL)